api/samples/CMakeLists.txt

```diff
@@ -224,21 +224,25 @@ configure_DynamoRIO_global(OFF ON)
 
 # FIXME i#4372: Add functioning tests for samples that go beyond  # NON-PUBLIC
 # sanity checks.                                                  # NON-PUBLIC
-add_sample_client(bbbuf       "bbbuf.c"         "drmgr;drreg;drx")
-add_sample_client(bbcount     "bbcount.c"       "drmgr;drreg;drx")
-add_sample_client(bbsize      "bbsize.c"        "drmgr;drx")
+if (NOT RISCV64) # FIXME i#3544: Port tests to RISC-V64
```

Comment:
If the idea is to exclude absolutely everything in this file, could we have a single return() instead to avoid cluttering the file with if() statements?